bot/seasons/evergreen/magic_8ball.py

```diff
@@ -0,0 +1,34 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Magic8ball:
+    """
+    A Magic 8ball command to respond to a users question.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+        with open(Path("bot", "resources", "evergreen", "magic8ball.json"), "r") as file:
+            self.answers = json.load(file)
+
+    @commands.command(name="8ball")
+    async def output_answer(self, ctx, question: list):
```

Comment:
Why is `question` being treated as a list here, rather than a string?

Take a look at the [Command parameters](https://discordpy.readthedocs.io/en/rewrite/ext/commands/commands.html#parameters) documentation, a keyword-only argument would probably be more appropriate here. Since this will currently only capture the first word of the user's input as `question`, this can lead to false positives on the length check if the question starts with a short word, e.g. `Do Androids Dream of Electric Sheep?`